synapse/cortex.py

```diff
@@ -1618,15 +1618,14 @@ async def getCoreQueue(self, iden):
         return self.multiqueue.status(iden)
 
     async def reqCoreQueue(self, iden):
-        info = await self.getCoreQueue(iden)
-        if info is None:
+        try:
+            return await self.getCoreQueue(iden)
+        except s_exc.NoSuchName:
             raise s_exc.NoSuchName(mesg=f'No queue with iden {iden}', iden=iden)
```

Comment:
Why are you doing this? Raising the same exception with different args doesn't make sense to me.

Also, see my comment above about `getCoreQueue()` should return None instead of raising an exception.